docs/organizations/managing-people.md

```diff
@@ -9,7 +9,10 @@ For **private repositories**, Codacy only analyzes commits from people in your C
 
     -   Committers who aren't part of your Git provider organization can't join your organization on Codacy app, but you should still add them to your Codacy organization to analyze their commits to private repositories.
 
-    -   In some Enterprise plans, Codacy automatically adds to the organization new people that commit to your private repositories and analyzes their commits. Also, Codacy updates your seats automatically when adding these new committers.
+    -   In some Enterprise plans, Codacy automatically manages people activity and seats usage for your organization:
+        -   Adds new people who commit to your private repositories and analyzes their commits.
+        -   Deactivates people who perform no commit or login for 90 consecutive days.
+        -   Updates your organization seats accordingly when adding or deactivating people.
 
 To list and manage the people in your organization, open your organization **Settings**, page **People**. This page also shows their last activity on Codacy:
 
```

Comment:
This admonition is becoming long and has a nested list (a red flag IMO).

How about breaking it up and contextualizing the new admonition under the paragraph below? This would separate it logically and contextualize it as a note, like the admonitions further down.

Thoughts?

```suggestion

To list and manage the people in your organization, open your organization **Settings**, page **People**. This page also shows their last activity on Codacy.

!!! note
    In some Enterprise plans, Codacy automatically manages people activity and seat usage for your organization:
    
    -   Adds new people who commit to your private repositories and analyzes their commits.
    -   Deactivates people who perform no commit or login for 90 consecutive days.
    -   Updates your organization seats accordingly when adding or deactivating people.

```

Reply:
Yes, I believe this works. Thanks for the suggestion! 🚀